security/remote_devices/docker-swarm/README.md

```diff
@@ -4,4 +4,4 @@ This is a proof-of-concept demonstrating how to run EdgeX in Docker Swarm using
 
 ## Build and Run
 
-Please see the detailed steps in "how-to guide" for Docker Swarm in documentation repository here: <https://github.com/edgexfoundry/edgex-docs/blob/master/docs_src/microservices/security/Ch-Docker-Swarm-SecuringDeviceServices.md>.
+Please see the detailed steps in "how-to guide" for Docker Swarm in documentation repository here: <https://github.com/edgexfoundry/edgex-docs/blob/master/docs_src/security/Ch-Docker-Swarm-SecuringDeviceServices.md>.
```

Comment:
Please refence the Docs PR in this PR's description.

Reply:
Updated PR description.